Pipfile

```
@@ -9,7 +9,7 @@ python_version = "3.11"
 [packages]
 api-insee = "1.5"
 celery = {version = "==5.4.0", extras = ["redis"]}
-django = "~=4.2.16"
+django = "~=4.2.17"
```

```suggestion
django = "~=4.2"
```

Je pense qu'on peut relâcher un peu la contrainte de version sur cette dépendance. De manière générale, les release de Django sont très stables. On peut se permettre de leur faire confiance.